### ISSUE TITLE
A1111 on Kaggle Terminated 

### ISSUE THREAD

**Issue by BarrenWardo** [FurkanGozukara/Stable-Diffusion#30]
Kaggle can't be used for Stable Diffusion or any WebUIs anymore.

https://www.kaggle.com/discussions/product-feedback/440296#2438857

Update : Tutorials/How-To-Use-Automatic1111-Web-UI-On-A-Free-Kaggle-Notebook-Like-Google-Colab.md

**Comment by FurkanGozukara** [comment]
Correct. Currently we are using another method and it is only shared on Patreon since we don't want it to be blocked too : https://www.patreon.com/posts/run-on-free-like-88714330

**Comment by 0xElessar** [comment]
Thank you for your work and knowledge sharing, @FurkanGozukara . But I really don't get why this mechanism was blocked. The platform is for AI research and this is exactly what we are doing.

**Comment by FurkanGozukara** [comment]
> Thank you for your work and knowledge sharing, @FurkanGozukara . But I really don't get why this mechanism was blocked. The platform is for AI research and this is exactly what we are doing.

i 100% agree with you

I even told this to the Kaggle team

They say you can use Google colab PRO :/ Google colab pro is much worse than RunPod

**Comment by jjiikkkk** [comment]
okay, then another question, this command
!yes ｜ apt install libgoogle-perftools-dev, too long, apt install.... is enough. right?

is USD 5 payable on a month at any time or each month?

**Comment by FurkanGozukara** [comment]
> > > > Correct. Currently we are using another method and it is only shared on Patreon since we don't want it to be blocked too : https://www.patreon.com/posts/run-on-free-like-88714330
> > > 
> > > 
> > > how much should I patreon and whether this method can be maintained for long on kaggle?
> > 
> > 
> > bronze 5$
> > well as long as you don't generate NSFW, super important, people are using every day for hours
> 
> and could you show me a link for this USD 5? I want to know if it should be paid for one month or each month?

you can pay download and cancel. it is subscription until you cancel it : https://www.patreon.com/checkout/SECourses?rid=9302127

**Comment by jjiikkkk** [comment]
> > > > > Correct. Currently we are using another method and it is only shared on Patreon since we don't want it to be blocked too : https://www.patreon.com/posts/run-on-free-like-88714330
> > > > 
> > > > 
> > > > how much should I patreon and whether this method can be maintained for long on kaggle?
> > > 
> > > 
> > > bronze 5$
> > > well as long as you don't generate NSFW, super important, people are using every day for hours
> > 
> > 
> > and could you show me a link for this USD 5? I want to know if it should be paid for one month or each month?
> 
> you can pay download and cancel. it is subscription until you cancel it : https://www.patreon.com/checkout/SECourses?rid=9302127

paid already

<html>
<body>


交易号0JG72306F96228228 | 交易号0JG72306F96228228 | 交易日期2023年10月2日 GMT+8 下午10:56:37 | 交易日期2023年10月2日 GMT+8 下午10:56:37
-- | -- | -- | --
交易号0JG72306F96228228
交易日期2023年10月2日 GMT+8 下午10:56:37

商家Patreonnoreply@patreon.com+1 0000000000 | 商家Patreonnoreply@patreon.com+1 0000000000
商家Patreonnoreply@patreon.com+1 0000000000


</body>
</html>交易号
[0JG72306F96228228](https://www.paypal.com/cgp/app-redirect?intent=xo_email_txn_details&src=RT001736&ref_id=0JG72306F96228228&url_clicked_desc=txn-id-hyperlink&v=1&utm_source=unp&utm_medium=email&utm_campaign=RT001736&utm_unptid=035709b2-6134-11ee-8f51-40a6b722824c&ppid=RT001736&cnac=C2&rsta=zh_XC%28zh-CN-xbord%29&cust=WT8KZWQEGDME6&unptid=035709b2-6134-11ee-8f51-40a6b722824c&calc=eccbc53398a54&unp_tpcid=email-receipt-member-sale-transaction-rt001736&page=main%3Aemail%3ART001736&pgrp=main%3Aemail&e=cl&mchn=em&s=ci&mail=sys&appVersion=1.206.0&xt=104038%2C127632)

交易日期
2023年10月2日 GMT+8 下午10:56:37

商家
Patreon
[noreply@patreon.com](mailto:noreply@patreon.com)
+1 0000000000

**Comment by jjiikkkk** [comment]
> > > > > Correct. Currently we are using another method and it is only shared on Patreon since we don't want it to be blocked too : https://www.patreon.com/posts/run-on-free-like-88714330
> > > > 
> > > > 
> > > > how much should I patreon and whether this method can be maintained for long on kaggle?
> > > 
> > > 
> > > bronze 5$
> > > well as long as you don't generate NSFW, super important, people are using every day for hours
> > 
> > 
> > and could you show me a link for this USD 5? I want to know if it should be paid for one month or each month?
> 
> you can pay download and cancel. it is subscription until you cancel it : https://www.patreon.com/checkout/SECourses?rid=9302127

my discord
virginia8347

**Comment by jjiikkkk** [comment]
> > > > > Correct. Currently we are using another method and it is only shared on Patreon since we don't want it to be blocked too : https://www.patreon.com/posts/run-on-free-like-88714330
> > > > 
> > > > 
> > > > how much should I patreon and whether this method can be maintained for long on kaggle?
> > > 
> > > 
> > > bronze 5$
> > > well as long as you don't generate NSFW, super important, people are using every day for hours
> > 
> > 
> > and could you show me a link for this USD 5? I want to know if it should be paid for one month or each month?
> 
> you can pay download and cancel. it is subscription until you cancel it : https://www.patreon.com/checkout/SECourses?rid=9302127

.... you do not reply to me, cry
